source/fundamentals/monitoring.txt

```diff
@@ -1,7 +1,435 @@
+                   "TopologyVersion": null,
+                   "Kind": 0,
+                   "WireVersion": null
+               },
+               ...
+           ],
+           "SetName": "...",
+           "Kind": 10,
+           "SessionTimeoutMinutes": 30,
+           "CompatibilityErr": null
+       },
+       "NewDescription": {
+           "Servers": [...],
+           "SetName": "...",
+           "Kind": 10,
+           "SessionTimeoutMinutes": 30,
+           "CompatibilityErr": null
+       }
+   }
+
+The ``Kind`` field in the server descriptions represents the type of a
+single server in a topology and can have the following values:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Value
+     - Description
+   * - ``0``
+     - Unknown instance
+   * - ``1``
+     - Standalone instance
+   * - ``2``
+     - Replica set member
+   * - ``6``
+     - Primary instance
+   * - ``10``
+     - Secondary instance
+   * - ``18``
+     - Arbiter instance
+   * - ``34``
+     - Replica set ghost, a member that cannot be queried
+   * - ``256``
+     - Mongos proxy instance
+   * - ``512``
+     - Load balancer instance
+
+TopologyOpeningEvent
+~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: bash
+   :copyable: false
+
+   *event.TopologyOpeningEvent
+   {
+       "TopologyID": "..."
+   }
+
+TopologyClosedEvent
+~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: bash
+   :copyable: false
+
+   *event.TopologyClosedEvent
+   {
+       "TopologyID": "..."
+   }
+
+ServerHeartbeatStartedEvent
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: bash
+   :copyable: false
+
+   *event.ServerHeartbeatStartedEvent
+   {
+       "ConnectionID": "...",
+       "Awaited": true
+   }
+
+ServerHeartbeatSucceededEvent
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: bash
+   :copyable: false
+
+   *event.ServerHeartbeatSucceededEvent
+   {
+    "DurationNanos": ...,
+       "Reply": {
+           "Addr": "...",
+           "Arbiters": null,
+           "AverageRTT": 0,
+           "AverageRTTSet": false,
+           "Compression": null,
+           "CanonicalAddr": "...",
+           "ElectionID": "...",
+           "HeartbeatInterval": 0,
+           "HelloOK": true,
+           "Hosts": [...],
+           "LastError": null,
+           "LastUpdateTime": "...",
+           "LastWriteTime": "...",
+           "MaxBatchCount": ...,
+           "MaxDocumentSize": ...,
+           "MaxMessageSize": ...,
+           "Members": [...],
+           "Passives": null,
+           "Passive": false,
+           "Primary": "...",
+           "ReadOnly": false,
+           "ServiceID": null,
+           "SessionTimeoutMinutes": 30,
+           "SetName": "...",
+           "SetVersion": 9,
+           "Tags": [...],
+           "TopologyVersion": {...},
+           "Kind": 6,
+           "WireVersion": {...}
+       },
+       "ConnectionID": "...",
+       "Awaited": true
+   }
+
+The ``Kind`` field in the server description represents the type of a
```

Comment:
```suggestion
The ``Kind`` field in the ``ServerHeartbeatSucceededEvent`` document represents the type of a
```

Reply:
N/A because restructuring to have one table.